apps/glusterfs/app_volume.go

```diff
@@ -246,8 +246,22 @@ func (a *App) VolumeDelete(w http.ResponseWriter, r *http.Request) {
 			return err
 		}
 
-		return nil
+		if !volume.Info.Block {
```

Comment:
Thanks @MohamedAshiqrh , I would like to see a design document for this feature as part of this patch. We can add it to doc/blockvolume.md or something. This doc should highlight (not to deeply) how Heketi is supporting this and how the internals work.